internal/repositories/github/service.go

```diff
@@ -369,7 +387,7 @@ func (r *repositoryService) pushReconcilerEvent(pbRepo *pb.Repository, projectID
 // returns DB PK along with protobuf representation of a repo
 func (r *repositoryService) persistRepository(
 	ctx context.Context,
-	githubRepo *github.Repository,
+	repoProperties *properties.Properties,
 	githubHook *github.Hook,
```

Comment:
we still need to get rid of the github hook if we want to make the code better reusable but most of the data we store into the pbRepository is not really used past registration..so we could probably get away with refactoring webhokManager to return a struct with a more minimal attribute set and just don't fill a lot of the attributes

Reply:
The github webhook installation will be a trait by itself. But that will be generalized as part of another body of work. This is fine for now.